src/app/OperationalSessionSetup.cpp

```diff
@@ -591,6 +601,8 @@ CHIP_ERROR OperationalSessionSetup::LookupPeerAddress()
         return CHIP_NO_ERROR;
     }
 
+    MATTER_LOG_METRIC_BEGIN(kMetricDeviceOperationalDiscovery);
```

Comment:
Similar question here: it looks like we can reach this point multiple times before ever hitting a `MATTER_LOG_METRIC_END(kMetricDeviceOperationalDiscovery)`, no?

Reply:
Yes, that is fine. Backend needs to handle this

Reply:
Yup